main.py

```diff
@@ -14,59 +15,56 @@
     parser.add_argument("--no-bio", help="Don't fetch biography", action="store_true")
     parser.add_argument("--no-album", help="Don't download pictures", action="store_true")
```

Comment:
the logic is broken here. I have run main.py as:

`python ./main.py --target emma-hix --no-album --no-video --log debug`

and it is downloading images.